server/routerlicious/packages/services-client/package.json

```diff
@@ -68,7 +68,7 @@
 		"debug": "^4.3.4",
 		"json-stringify-safe": "^5.0.1",
 		"jsrsasign": "^10.5.25",
-		"jwt-decode": "^3.0.0",
+		"jwt-decode": "^4.0.0",
```

Comment:
@znewton Any concerns about upgrading jwt-decode in routerlicious?

Reply:
Docs for this upgrade indicate there isn't much changed: https://github.com/auth0/jwt-decode/releases/tag/v4.0.0

Its just dropping node 14 and 16 support, and improving CJS/ESM stuff.